.github/workflows/deploy-frontends-to-production.yml

```diff
@@ -225,6 +226,7 @@ jobs:
           --build-arg REACT_APP_BASE_AIRQLOUDS_URL=${{ secrets.WEBSITE_REACT_APP_BASE_AIRQLOUDS_URL }} \
           --build-arg REACT_APP_BASE_NEWSLETTER_URL=${{ secrets.WEBSITE_REACT_APP_BASE_NEWSLETTER_URL }} \
           --build-arg REACT_APP_WEBSITE_BASE_URL=${{ secrets.WEBSITE_PROD_REACT_APP_WEBSITE_BASE_URL }} \
+          --build-arg REACT_APP_AUTHORIZATION_TOKEN=${{ secrets.WEBSITE_PROD_REACT_APP_AUTHORIZATION_TOKEN }} \
```

Comment:
[add REACT_APP_AUTHORIZATION_TOKEN to build pipeline](https://github.com/airqo-platform/AirQo-frontend/pull/813/commits/c758612860fa93ab7aa1d5462de2f33a15a5523f)